project_scrum/views/project_scrum_view.xml

```diff
@@ -0,0 +1,1138 @@
+            <field name="model">project.project</field>
+            <field name="priority">30</field>
+            <field name="inherit_id" ref="project.edit_project" />
+            <field name="arch" type="xml">
+
+                <button name="attachment_tree_view" position="before">
+                    <button class="oe_inline oe_stat_button"
+                        type="action" icon="fa-files-o"
+                        attrs="{'invisible': [('use_scrum', '=', False)]}"
+                        name="%(action_ps_us_current)d">
+                        <field name="user_story_count" widget="statinfo" />
+                    </button>
+                    <button class="oe_inline oe_stat_button"
+                        type="action" icon="fa-tasks"
+                        attrs="{'invisible': [('use_scrum', '=', False)]}"
+                        name="%(action_ps_sprint_current)d">
+                        <field name="sprint_count" widget="statinfo" />
+                    </button>
+                    <button class="oe_inline oe_stat_button"
+                        type="action" icon="fa-files-o"
+                        attrs="{'invisible': [('use_scrum', '=', False)]}"
+                        name="%(action_ps_tc_current)d">
+                        <field name="test_case_count" widget="statinfo" />
+                    </button>
+                    <button class="oe_inline oe_stat_button"
+                        type="action" icon="fa-files-o"
+                        attrs="{'invisible': [('use_scrum', '=', False)]}"
+                        name="%(action_ps_meeting_current)d">
+                        <field name="meeting_count" widget="statinfo" />
+                    </button>
+                </button>
+
+
+                <field name="use_tasks" position="before">
+                    <field name="use_scrum" class="oe_inline" />
+                    <label for="use_scrum" string="Use Scrum" />
+                </field>
+
+
+                <field name="partner_id" position="after">
+                    <group>
+                        <label for="manhours" string="Expected Hours"
+                            attrs="{'invisible': [('use_scrum', '=', False)]}" />
+                        <label for="manhours"
+                            attrs="{'invisible': [('use_scrum', '=', True)]}" />
+                        <field name="manhours" nolabel="1" />
+
+                    </group>
+
+                    <field name="description" />
+
+                </field>
+
+            </field>
+        </record>
+        <!--access to sprint and user story from task form view -->
+        <record id="view_ps_sprint_task_form2" model="ir.ui.view">
+            <field name="name">project.scrum.task.form</field>
+            <field name="model">project.task</field>
+            <field name="inherit_id" ref="project.view_task_form2" />
+            <field name="priority">99</field>
+            <field name="arch" type="xml">

```

Comment:
I think we need a separation of scrum tasks and non-scrum tasks. So better make this a primary view, and add a menu that forces this view. Do the same with actions that open tasks of a scrum project.

Alternatively, change this view such that it doesn't change anything for non-scrum tasks.